hack/cluster-sync.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+set -e
+
+TEMP_FILE=$(mktemp -p /tmp -t kubevirt.deploy.XXXX)
+
+trap 'rm -f $TEMP_FILE' EXIT SIGINT
+
+function main() {
+    ./hack/cluster-clean.sh --skip-wait >$TEMP_FILE 2>&1 &
+    CLEAN_PID=$!
+
+    make cluster-build
+
+    echo "waiting for cluster-clean to finish"
+    if ! wait $CLEAN_PID; then
+        echo "cluster-clean failed, output was:"
+        cat $TEMP_FILE
```

Comment:
This should happen also for the "regular" path (of no errors).

Reply:
No need to imo, it just spam the output, no interesting info there

Reply:
It exists now, and if you think it is not needed, then send another PR with this reasoning (which may fit many other outputs as well).

Reply:
I think its ok to have it in this PR, it fit in the scope of running the clean in the background imo

Reply:
The side-effect of this parallel run is the lost of the logs, which may or may not be fine. You have said nothing about it in the commit and it may pass un-noticed. I pointed this out here so it will not be missed.
Reasoning to drop logging is something that should be visible.

Reply:
Adding to the commit message